docs/development/workflow/development_workflow.rst

```diff
@@ -31,12 +31,13 @@ development version or stable version is the active one.
 up. It is well worth your time.
 
 Not sure what your first contribution should be? Take a look at the `Astropy
-issue list`_ and grab one labeled "package-novice". These issues are the
-most accessible ones if you are not familiar with the Astropy source
-code. Issues labeled as "effort-low" are expected to take a few hours (at
-most) to address, while the "effort-medium" ones may take a few days. The
-developers are friendly and want you to help, so don't be shy about asking
-questions on the `astropy-dev mailing list`_.
+issue list`_ and grab one labeled `"package-novice" <https://github.com/astropy/astropy/issues?q=is%3Aissue+is%3Aopen+label%3Apackage-novice>`_.
+These issues are the most accessible ones if you are not familiar with the
+Astropy source code. Issues labeled as `"effort-low" <https://github.com/astropy/astropy/issues?q=is%3Aissue+is%3Aopen+label%3Aeffort-low>`_
+are expected to take a few hours (at most) to address, while the
+`"effort-medium" <https://github.com/astropy/astropy/issues?q=is%3Aissue+is%3Aopen+label%3Aeffort-medium>`_
+ones may take a few days. The developers are friendly and want you to help, so
+don't be shy about asking questions on the `astropy-dev mailing list`_.
```

Comment:
It is fine if you don't want to add a link here but in that case, probably should remove the trailing `_`.
```suggestion
don't be shy about asking questions on the `astropy-dev mailing list`.
```

Reply:
Not sure what happened with previous comment or if you missed it, but the `_` should remain.  `docs/conf.py` has the links specified and both `stable` and `latest` provide the proper link.

Reply:
Ah, okay! Thanks for the clarification.